test/index.js

```diff
@@ -1,19 +1,20 @@
-const { expect } = require('chai');
-const P = require('../src/');
+// const { expect } = require('chai');
```

Comment:
Why is this all commented out?

Reply:
whoops forgot to delete, this has been replaced by a new test in `proofPoints.js`